src/warehouse/warehouse.ts

```diff
@@ -0,0 +1,35 @@
+import { env } from 'decentraland-commons'
+
+const WAREHOUSE_URL: string = env.get('WAREHOUSE_URL')
+const WAREHOUSE_TOKEN: string = env.get('WAREHOUSE_TOKEN')
+const WAREHOUSE_CONTEXT_PREFIX: string = env.get('WAREHOUSE_CONTEXT_PREFIX')
```

Comment:
I think it'd be a good idea to add a throw here explaining that the variables are missing, as they don't have a default value and the ToS eendpoint fails silently https://github.com/decentraland/builder-server/pull/251/files#diff-ea5b87ec142e269f5ad70b6154f928ebe98878ba0656ae127be867ea1a5513d0R114

what do you think?